public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -132,6 +141,7 @@ export class UnConnectedExploreToolbar extends PureComponent<Props, {}> {
 
   stopLive = () => {
     const { exploreId } = this.props;
+    this.pauseLive();
```

Comment:
This is needed so that logs do not get updated and reordered during transition as we still show live logs component while `isLive === false`. Is also duplicated and I would like to move the whole stop/resume/pause functionality into a hook (need to update react-redux first) or hoc did not want to inflate this PR.